README.md

```diff
@@ -63,11 +63,11 @@ To build and test in Visual Studio, open
 Alternatively, you can build REEF.NET from a developer command line
 via:
 
-    msbuild .\lang\cs\Org.Apache.REEF.sln
+    msbuild .\lang\cs\Org.Apache.REEF.sln /p:Configuration="Debug" /p:Platform="x64"
```

Comment:
They can be default if you have corresponding environment variables defined. For me, I don't have `Configuration` variable, and `Platform` was set to BWS for some reason, so I had to provide both explicitly even for existing build command. 

I can modify test project to default to Debug|x64 if nothing is provided, but some people will still need to pass params explicitly